readme.md

```diff
@@ -1,37 +1,52 @@
-# How to use this example bootstrap
+# Getting up and running
 
-This site bootstrap is designed to be used with [Varying Vagrants Vagrant](https://github.com/Varying-Vagrant-Vagrants/VVV/) and a WordPress single site, the code for which is stored as a monolithic (or submoduled, probably) Git(Hub) repo.
+Step 0: Read the docs
 
-# Initial configuration
+VVV2 [Adding a New Site](https://varyingvagrantvagrants.org/docs/en-US/adding-a-new-site/)
 
-The following steps allow you to adapt this template to your own VIP Go client repository:
+Step 1: Ensure you have a `vvv-custom.yml` file
 
-1. Run a search and replace for `site-name.dev` to whatever the subdomain for your development site will be
-2. Run a search and replace for `site_name` to whatever the database name for your development site will be
-3. Run a search and replace for `Site Name` to whatever the human readable name for your development site will be
+If you don't already have a `vvv-custom.yml` file, copy `vvv-config.yml` to create this.
 
-You can now skip to "Getting up and running", or you want to package these instructions for other developers in your team by following these instructions:
+Step 2: Add your site to  `vvv-custom.yml`
 
-1. Remove these "initial configuration" instructions, leaving the "Development environment bootstrap" heading and everything below it
-2. Test everything works as expected in a [VVV](https://github.com/Varying-Vagrant-Vagrants/VVV/) context
-3. Copy or `git push` to a new repo or new branch in an existing repo
-4. Point people towards the `readme.md` in the repo you pushed to, so they can get going
+Add a site block into `vvv-custom.yml`, like this:
 
-# Getting up and running
+``` yml
+  name-your-site-here: 
+    repo: https://github.com/YourOrg/your-repo.git
+    branch: master
+    hosts: 
+     - name-your-site-here.local
```

Comment:
it's actually possible to dynamically add hosts. The init provisioner script runs before the nginx stuff, so you can use a template nginx, pull the hosts out of the config, fill in the template and put an actual nginx file where VVV expects, and it'll all get picked up and wrangled by the provisioners. If you can configure domain mapping too then you can put anything you want in the hosts file and it'll work out of the box on provision. Loreleis custom site template used in the main VVV org does this